python/cuxfilter/charts/panel_widgets/plots.py

```diff
@@ -257,20 +237,12 @@ def initiate_chart(self, dashboard_cls):
         """
         initiate chart on dashboard creation
         """
-        if type(dashboard_cls._cuxfilter_df.data) == dask_cudf.core.DataFrame:
-            self.min_value = int(
-                dashboard_cls._cuxfilter_df.data[self.x].min().compute()
-            )
-            self.max_value = int(
-                dashboard_cls._cuxfilter_df.data[self.x].max().compute()
-            )
-        else:
-            self.min_value = int(
-                dashboard_cls._cuxfilter_df.data[self.x].min()
-            )
-            self.max_value = int(
-                dashboard_cls._cuxfilter_df.data[self.x].max()
+        self.min_value, self.max_value = (
+            int(i)
+            for i in self._get_min_max(
+                dashboard_cls._cuxfilter_df.data, self.x
```

Comment:
I usually like black formatting but this one took a second decipher with the short line length.  FWIW I would suggest `x` as a variable name for a value that is not an integer (and there are not integers *to start*). 

But maybe this would look clearer under black formatting?
```
min, max = self._get_min_max(dashboard_cls._cuxfilter_df.data, self.x)
self.min_value = int(min)
self.max_value = int(max)
```

Reply:
fixed